changelogs/fragments/441-system-downward-compatibility-fix.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - system - Catch the ConnectionError exception caused by unconditional fetching of auditd and ip loadshare hash algorithm configuration fetches and return empty configuration instead of allowing the uncaught exception to abort all "system" operations on SONiC images older than version 4.4.0 (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/434).
```

Comment:
```suggestion
  - sonic_system - Catch the ConnectionError exception caused by unconditional fetching of auditd and ip loadshare hash algorithm configuration and return empty configuration instead of allowing the uncaught exception to abort all "system" operations on SONiC images older than version 4.4.0 (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/434).
```

Reply:
Thanks for catching that grammar error in the description and the missing "sonic_" prefix on the component name.

I will fix this, but am not committing it "as is" because I see that I also forgot to change the copied PR number from "434" to "441".

Reply:
Fixed.